src/coq_module.ml

```diff
@@ -0,0 +1,50 @@
+(* This file is licensed under The MIT License *)
```

Comment:
I'm not sure this header is necessary given DCO. Someone from Jane Street should comment on this :)

Reply:
(same for all similar headers)

Reply:
I thought it would be safer to explicitly state the license :) No problem doing whatever is needed in this area.